packages/components/src/components/icon/icon-styles.ts

```diff
@@ -65,25 +65,13 @@ const filter: Record<Theme, Record<Exclude<IconColor, IconColorDeprecated | 'inh
   },
 };
 
-const forceRerenderAnimationStyle = {
-  '0%': {
-    transform: 'rotateZ(0)',
-  },
-  '100%': {
-    transform: 'rotateZ(0)',
-  },
-};
-const keyFramesLight = 'rerender-light';
-const keyFramesDark = 'rerender-dark';
-
 export const getComponentCss = (
   color: Exclude<IconColor, IconColorDeprecated>,
   size: TextSize,
   theme: Theme
 ): string => {
   const isColorInherit = color === 'inherit';
   const isSizeInherit = size === 'inherit';
-  const isDark = isThemeDark(theme);
 
   return getCss({
```

Comment:
is it still working when the animation hack is removed?

Reply:
<img width="966" alt="Screenshot 2023-03-28 at 16 03 03" src="https://user-images.githubusercontent.com/30120450/228263670-2a08739b-4a3f-470d-89d5-2ff7cb7eb358.png">
It does not work :-(